src/main/java/pro/sky/telegrambot/models/NotificationTask.java

```diff
@@ -0,0 +1,44 @@
+package pro.sky.telegrambot.models;
+
+import lombok.*;
+import org.hibernate.Hibernate;
+
+import javax.persistence.*;
+import java.time.LocalDateTime;
+import java.util.Objects;
+
+@Entity
+@Table(name = "notification_task_name", schema = "schema_bot")
+@Getter
+@Setter
```

Comment:
изначально так и было, идея вот что написала:
Using @Data for JPA entities is not recommended. It can cause severe performance and memory consumption issues.